README.md

```diff
@@ -22,6 +22,23 @@ More commands are available, for reference, see:
 $ make help
 ```
 
+## Advanced configuration
+
+### Health checks
+
+Our docker image embeds the [lightship](https://github.com/gajus/lightship)
+application as an etherpad-lite plugin. It provides health check status
+endpoints to a running etherpad instance. The following environment variable
+can be injected in the container to configure its behavior:
+
+- `EP_LIGHTSHIP_DETECT_KUBERNETES`: a boolean-like string to choose whether
+  `lightship` should detect its environment and assign a random port to listen
+  to while not in k8s environment (default: `false`),
```

Comment:
Why this value is false by default, unlike the default value defined in lightship ?
It does not anything harmful (see [here](https://github.com/gajus/lightship/blob/master/src/utilities/isKubernetes.js)).

What do you think about adding another variable `EP_LIGHTSTRIP_ENABLED` (default : `false`) to disable totally this feature ?

Reply:
> Why this value is false by default, unlike the default value defined in lightship ?
It does not anything harmful (see here).

While running in a docker-compose context, as `KUBERNETES_SERVICE_HOST` is not defined, the application will randomly choose an available local port (and defining the `EP_LIGHTSHIP_PORT` has no effect on it). I find it more relevant to use an explicit / predictable port by default.

Reply:
> What do you think about adding another variable EP_LIGHTSTRIP_ENABLED (default : false) to disable totally this feature ?

You mean disable this plugin?

Reply:
> While running in a docker-compose context, as `KUBERNETES_SERVICE_HOST` is not defined, the application will randomly choose an available local port (and defining the `EP_LIGHTSHIP_PORT` has no effect on it). I find it more relevant to use an explicit / predictable port by default.

Hmm yes, you are completely right.

Reply:
> > What do you think about adding another variable EP_LIGHTSTRIP_ENABLED (default : false) to disable totally this feature ?
> 
> Do you mean disable this plugin?

Yes I mean not loading this plugin or easier :  doing nothing in `src/plugins/ep_lightstrip/server.js`.
Since this feature is not needed by everyone, I think it's a good thing to be able to disable it.

Reply:
FWIU the etherpad plugins credo is: if it is installed, it is used! The only way to inactivate a plugin is to uninstall it.